modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,95 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-0_{context}"]
+= Release notes for {gitops-title} 1.12.0
+
+{gitops-title} 1.12.0 is now available on {OCP} 4.12, 4.13, and 4.14.
```

Comment:
are we not doing 4.15? It's released now...

Reply:
It is released and I can add it too, pending further confirmation from you. I erred on the side of caution and did not include it. Please confirm if this release information can be included.

Reply:
It seems like the announcement for 4.15 GA happens on March 19th so I believe e would need to make that change later

Reply:
cc: @Srivaralakshmi

Reply:
Should I add that information now @varshab1210 or wait for the 19th although the release is going on March 14th. This is indeed a slight grey area. 

Tagging @iam-veeramalla and @reginapizza in this comment to get their views as well.

Reply:
> It seems like the announcement for 4.15 GA happens on March 19th so I believe e would need to make that change later

Hi @varshab1210 - As today is March 19th, I am including information about 4.15 in our RN. Please confirm if you are fine with this change. Thanks.